jestconfig.js

```diff
@@ -18,5 +18,10 @@ module.exports = {
     '\\.(svg|jpg|png|md)$': '<rootDir>/__mocks__/fileMock.js',
     '\\.(css|scss)$': 'identity-obj-proxy',
   },
+  moduleDirectories: [
+    'aggregated-translations',
+    './packages/terra-i18n/tests/jest/aggregated-translations',
```

Comment:
This allows Jest to resolve the generated `intlLoaders.js` and `translationsLoader.js` files from the aggregated-translations pre-build tool inside of the terra-i18n tests.